### DISCUSSION TITLE
MISP idstools rules not being pushed to forward nodes

### DISCUSSION THREAD

**Discussion by polivia** [Security-Onion-Solutions/securityonion#1960] [Unsupported Versions] [answered]
Manager:
----------
[root@so-manager ~]# ls -ltr /opt/so/saltstack/local/salt/idstools/misp.rules
-rw-r--r--. 1 socore socore 96771296 Nov 17 06:01 /opt/so/saltstack/local/salt/idstools/misp.rules

[root@so-manager ~]# ls -ltr /opt/so/saltstack/local/salt/zeek/policy/intel/misp-intel.dat
-rw-r--r--. 1 socore socore 68669632 Nov 17 06:02 /opt/so/saltstack/local/salt/zeek/policy/intel/misp-intel.dat

Sensor:
-------
[root@so-sensor01 ~]# ls -ltr /opt/so/conf/suricata/rules/misp.rules
-rw-r--r--. 1 suricata suricata 0 Nov  5 15:48 /opt/so/conf/suricata/rules/misp.rules

[root@so-sensor01 ~]# ls -ltr /opt/so/conf/zeek/policy/intel/misp-intel.dat
-rw-r--r--. 1 zeek socore 68669632 Nov 17 06:15 /opt/so/conf/zeek/policy/intel/misp-intel.dat

**Comment by weslambert** [comment]
I haven't had a chance to look back into this just yet -- still need to perform testing on my side.  Thanks for the feedback!

**Comment by weslambert** [comment, marked as answer]
I still need to investigate this -- will have a look if I have time today.

**Reply by polivia** [reply]
Any update on this?

**Reply by polivia** [reply]
This seems to be working now.  I'm guessing a soup update at some point fixed it.
